java/java.source.base/src/org/netbeans/modules/java/source/TreeShims.java

```diff
@@ -227,6 +229,33 @@ public static ExpressionTree getGuardedExpression(Tree node) {
             throw TreeShims.<RuntimeException>throwAny(ex);
         }
     }
+	
+	public static List<DocTree> getSnippetDocTreeAttributes(DocTree node) {
+        try {
+            Class gpt = Class.forName("com.sun.source.doctree.SnippetTree"); //NOI18N
+            return isJDKVersionRelease18_Or_Above()
+                    ? (List<DocTree>)gpt.getDeclaredMethod("getAttributes").invoke(node)  //NOI18N
+                    : null;
```

Comment:
wouldn't returning empty list be more apropriate?

Reply:
Addressed in latest commit(https://github.com/apache/netbeans/commit/bb448a61294adecae9571d7d9e02279c15e7c1d9)